spec/lib/secure_headers/view_helpers_spec.rb

```diff
@@ -64,6 +70,16 @@ def content_tag(type, content = nil, options = nil, &block)
     "<#{type}#{options}>#{content}</#{type}>"
   end
 
+  def javascript_include_tag(source, options = {})
+    content_tag(:script, nil, options.merge(src: source))
+  end
+
+  alias_method :javascript_pack_tag, :javascript_include_tag
```

Comment:
What is the reason for the alias? Can we accomplish the same thing without the metaprogramming?

Reply:
We could, but for the purpose of this test the method is identical. Would you prefer that?